examples/spacex/app/rsc/page.tsx

```diff
@@ -8,14 +9,11 @@ import styles from './page.module.css'
 import { PageNumbers } from '@/components/PageNumbers'
 import { LaunchDetails } from '@/components/LaunchDetails'
 
-const { getClient } = registerClient(() =>
-  createClient({
-    url:
-      process.env.NODE_ENV === 'production'
-        ? 'https://spacex-fuse.vercel.app/api/fuse'
-        : 'http://localhost:3000/api/fuse',
-  }),
-)
+const files = require.context('../../types', true, /\.ts$/)
```

Comment:
Somehow we need to remove the `require.context` to not make this optimisation feel like a hurdle